src/App.vue

```diff
@@ -1,25 +1,33 @@
 <template>
   <div
-    v-if="config"
     id="app"
+    v-if="config"
     :class="[
-      `theme-${config.theme}`,
-      `page-${currentPage}`,
-      isDark ? 'dark' : 'light',
+      'theme-' + config.theme,
+      'page-' + currentPage,
+      isDark ? 'is-dark' : 'is-light',
       !config.footer ? 'no-footer' : '',
     ]"
   >
-    <DynamicTheme v-if="config.colors" :themes="config.colors" />
+    <DynamicTheme :themes="config.colors" />
+
     <div id="bighead">
       <section v-if="config.header" class="first-line">
         <div v-cloak class="container">
           <div class="logo">
             <a href="#">
-              <img v-if="config.logo" :src="config.logo" alt="dashboard logo" />
+              <img
+                v-if="config.logo"
+                :src="config.logo"
+                alt="dashboard logo"
+              />
```

Comment:
Unnecessary change. If the linter is ok with it, it does not have to be changed. Did you at least run `pnpm lint`?